packages/stark-testing/rollup.config.js

```diff
@@ -1,20 +1,16 @@
 "use strict";
 
-const resolve = require("rollup-plugin-node-resolve");
-const commonjs = require("rollup-plugin-commonjs");
+const commonData = require("../rollup.config.common-data.js"); // common configuration between environments
 
 module.exports = {
-	//export default {
 	input: "../../dist/packages-dist/stark-testing/karma.conf.typescript.js",
+	external: commonData.external,
+	plugins: commonData.plugins,
```

Comment:
Do we really want to include globals and all the rollup plugins here? Normally the stark-testing package just contains scripts and npm dependencies...

Reply:
I went from the assumption that we might later also add utility code in there. But that's not for now indeed and when we get there we'll probably rework that config